common/src/main/java/io/pravega/common/util/BitConverter.java

```diff
@@ -226,4 +226,35 @@ public static long readLong(byte[] source, int position) {
                 | (source[position + 6] & 0xFF) << 8
                 | (source[position + 7] & 0xFF);
     }
+
+    /**
+     * Writes the given 64-bit Unsigned Long to the given byte array at the given offset. This value can then be
+     * deserialized using {@link #readUnsignedLong}. This method is not interoperable with {@link #readLong}.
+     *
+     * The advantage of serializing as Unsigned Long (vs. a normal Signed Long) is that the serialization will have the
+     * same natural order as the input value type (i.e., if compared using a lexicographic bitwise comparator such as
+     * ByteArrayComparator, it will have the same ordering as the typical Long type).
+     *
+     * @param target The byte array to write to.
+     * @param offset The offset within the byte array to write at.
+     * @param value  The (signed) value to write. The value will be converted into the range [0, 2^64-1] before
```

Comment:
Could you update the javadoc here to reflect the simplified logic?

Reply:
done